src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftRewriter.kt

```diff
@@ -183,111 +177,6 @@ public open class RedshiftRewriter(val onProblem: ProblemCallback) : PlanRewrite
-     * a meta "EXPAND" set to true.
-     *
-     * @param steps
-     * @return
-     */
-    private fun StructType.exclude(steps: List<Rel.Op.Exclude.Step>): StaticType {
-        val step = steps.first()
-        val output = fields.mapNotNull { field ->
-            val newField = if (steps.size == 1) {
-                // excluding at current level
-                null
-            } else {
-                // excluding at a deeper level
-                val k = field.key
-                val v = field.value.exclude(steps.drop(1))
-                StructType.Field(k, v)
-            }
-            when (step) {
-                is Rel.Op.Exclude.Step.StructField -> {
-                    if (step.symbol.isEquivalentTo(field.key)) {
-                        newField
-                    } else {
-                        field
-                    }
-                }
-                is Rel.Op.Exclude.Step.StructWildcard -> newField
-                else -> field
-            }
-        }
-        val newMetas = this.metas.toMutableMap()
-        newMetas["EXPAND"] = true
-        return this.copy(fields = output, metas = newMetas)
-    }
-
-    /**
-     * Applies exclusions to collection element type.
-     *
-     * Note: this function should not actually be called since collection index and wildcard exclusion is not supported
-     * for Redshift transpilation.
-     *
-     * @param steps
-     * @return
-     */
-    private fun CollectionType.exclude(steps: List<Rel.Op.Exclude.Step>): StaticType {
-        var e = this.elementType
-        when (steps.first()) {
-            is Rel.Op.Exclude.Step.CollIndex -> {
-                if (steps.size > 1) {
-                    e = e.exclude(steps.drop(1))
-                }
-            }
-            is Rel.Op.Exclude.Step.CollWildcard -> {
-                if (steps.size > 1) {
-                    e = e.exclude(steps.drop(1))
-                }
-                // currently no change to elementType if collection wildcard is last element; this behavior could
-                // change based on RFC definition
-            }
-            else -> {
-                // currently no change to elementType and no error thrown; could consider an error/warning in
-                // the future
-            }
-        }
-        return when (this) {
-            is BagType -> this.copy(e)
-            is ListType -> this.copy(e)
-            is SexpType -> this.copy(e)
-        }
-    }
-
-    /**
-     * Compare an identifier to a struct field; handling case-insensitive comparisons.
-     *
-     * @param other
-     * @return
-     */
-    private fun Identifier.Symbol.isEquivalentTo(other: String): Boolean = when (caseSensitivity) {
-        Identifier.CaseSensitivity.SENSITIVE -> symbol == other
-        Identifier.CaseSensitivity.INSENSITIVE -> symbol.equals(other, ignoreCase = true)
-    }
```

Comment:
(self-review) Copied over to `src/main/kotlin/org/partiql/scribe/ExcludeBindings.kt`